components/ldap.rst

```diff
@@ -74,6 +74,19 @@ distinguished name (DN) and the password of a user::
 
     When the LDAP server allows unauthenticated binds, a blank password will always be valid.
 
+You can also use the :method:`Symfony\\Component\\Ldap\\Ldap::sasl_bind` method
+for binding to an LDAP server using `SASL`_::
+
```

Comment:
The method in Symfony is called saslBind() and not sasl_bind()

Reply:
Thanks! Fixed.